### DISCUSSION TITLE
Diabetes Retinopathy App - Pytorch/Tensorflow , Flask, HTML, CSS, JS

### DISCUSSION THREAD

**Discussion by aviprayag** [mrdbourke/pytorch-deep-learning#866] [Ideas]
I am glad to meet you Mr. Daniel Bourke. I'm from India and I love programming **CNN**(s) in Pytorch/Tensorflow modules. I would like to build an app with you if can buy some time for me so i would really appreciate it sir. And I've learned Tensorflow from your Udemy's Tensorflow Mastery Course. And I'm learning Pytorch from your Pytorch course so please i want to work with you. I would be more luckier than i was so thankyou Mr. Daniel Bourke.

**Comment by LuluW8071** [comment]
I have done some similar project to this. The PyTorch Modules is siimilar while you just need to change images for train and test. You can also contact me on my gmail or discord (click my profile) if you need any help. :)

## [Ocular-Eye-Disease-Classification](https://github.com/LuluW8071/Ocular-Eye-Disease-Classification/blob/main/Multiple_Eye_Disease_Detection.ipynb)

**Reply by aviprayag** [reply]
Can u work with me?

**Reply by LuluW8071** [reply]
Sure

**Reply by aviprayag** [reply]
so how?

**Reply by LuluW8071** [reply]
Send details on email : [Mail](mailto:shashinmaharjan@gmail.com)
